auth/auth_user.py

```diff
@@ -21,8 +27,7 @@ def login_page(cls):
 		cls.phantom_set.update([phantom])
 
 	@classmethod
-	def logout_page(cls,email,username,allocation):
-
+	def logout_page(cls,email,username,allocation,connect):
```

Comment:
shouldn't need the 'connect' param here as we can figure out whether or not the allocation is connecting via the status of the allocation